tests/components/analytics/test_analytics.py

```diff
@@ -19,11 +20,13 @@
 from homeassistant.setup import async_setup_component
 
 MOCK_UUID = "abcdefg"
+MOCK_VERSION = "1970.1.0"
+MOCK_NIGHTLY_VERSION = "1970.1.0.dev19700101"
 
 
 async def test_no_send(hass, caplog, aioclient_mock):
     """Test send when no prefrences are defined."""
-    aioclient_mock.post(ANALYTICS_ENDPOINT_URL, status=200)
+    aioclient_mock.post(ANALYTICS_ENDPOINT_URL_DEV, status=200)
```

Comment:
Will the tests pass on master when we release, or do we need to patch `HA_VERSION` everywhere?

Reply:
Ups.. no it will not, so yeah, this needs to be patched everywhere, I'll make an analytics fixture that can be reused in the tests that does that